### PR TITLE
Updated hoot logo

### PR DESCRIPTION
**Pull request by rmrice**
screenshot from 2017-01-26 09 57 28

### CODE REVIEW

**Comment by rmrice** [comment]
Doesn't work when deployed on hoot-server. Investigating.

File: js/hoot/view/utilities/Utilities.js [hunk 1]
```diff
@@ -173,34 +173,15 @@ Hoot.view.utilities = function (context){
...
+        .classed('point', true)
         .attr('href', '#version')
         .append('img')
-        .classed('col1 cursor', true)
-        .attr('src', 'img/hoot_logo_icon.png')
-        .attr('height', '60px');
-        labelContainer.append('div')
-        .classed('inline dark strong hoot_label cursor col2', true)
-        .html('Hootenanny')
+        .attr('src', context.asset('dist/img/logo/hoot_logo_update.png'))
```
Comment (brianhatchl) [inline js/hoot/view/utilities/Utilities.js]: I don't think you want the 'dist' in the path here.  That's what context.asset( ) adds if necessary.

Does this work when deployed via hootserver and tomcat?

Comment (rmrice) [inline js/hoot/view/utilities/Utilities.js]: No, the I discovered that the img src has to be defined in the css. It should work with the latest commits.

**Comment by rmrice** [comment]
This should work now.

**Pull request merged**